scripts/extract-databases.js

```diff
@@ -6,10 +6,6 @@ const path = require('path')
 
 fs.ensureDirSync('dist/shared')
 
-const customBundles = fs.readJSONSync('dist/shared/custom-bundles.json')
-const customTemplates = fs.readJSONSync('dist/shared/custom-templates.json')
-const publishedTemplates = fs.readJSONSync('dist/shared/published-templates.json')
-
 // NOTE: need to extract bundles before templates
 const files = ['bundles', 'templates-v2', 'symbols-v2', 'funders']
```

Comment:
Do we need symbols and funders here? we should also remove the db files.

Reply:
I checked we don't use them at all, removed